### ISSUE TITLE
add the option to disable 5 star burst cinematic 

### ISSUE THREAD

**Issue by casg1** [Akebi-Group/Akebi-GC#381]
option to make 5-star burst look like 4-star burst with no annoying cinematic, please :)

**Comment by lunaticwhat** [comment]
good idea. @Taiga74164 its very ez do it pls uwu

**Comment by lunaticwhat** [comment]
> good idea. @Taiga74164 its very ez do it pls uwu

ok nvm im sorry :copium:

**Comment by Kalip514** [comment]
I agree to this...please make a [disable] of the [burst action camera]

**Comment by EchoesFromBeyond** [comment]
I say hell to the YEAH to this. I have always wanted it.

**Comment by Taiga74164** [comment]
there is a simple way to disable it but the one we found doesn't let you move the camera around during the burst animation (

**Comment by Kalip514** [comment]
Damm.....shame really...

**Comment by casg1** [comment]
> there is a simple way to disable it but the one we found doesn't let you move the camera around during the burst animation (

Its ok, it doesn't really matter that u can't look around. As long as it disables it.
I don't want to have a seizure everytime i use an ult (it also looks cooler and less cringy)
